config/environments/test.rb

```diff
@@ -52,6 +52,12 @@
     Bullet.add_whitelist :type => :n_plus_one_query,
                          :class_name => 'BusinessUnit',
                          :association => :moved_to_unit
+    Bullet.add_whitelist :type => :n_plus_one_query,
+                         :class_name => 'Case::SAR::Offender',
+                         :association => :responder_assignment
+    Bullet.add_whitelist :type => :n_plus_one_query,
+                         :class_name => 'Case::SAR::OffenderComplaint',
+                         :association => :responder_assignment
```

Comment:
just for my knowledge, what do the above do? :)

Reply:
Is this allowing n+1 queries? Seems like this would be a bad thing to do... 🤔

Reply:
So, Bullet is a gem that we added to the project to track down slow queries  - it makes suggestions about how to fix n+1 query issues that it detects. The problem is that when we have complex ActiveRecord scopes like the ones in this app, its recommendations about how to fix it (typically using `include`) aren't actionable. This seems to have been consistent with previous developers, because there are a bunch of whitelisted cases in the app. I did spend some time experimenting to see if I could make it happy but couldn't find the magic incantation, so gave up and added two more to the list above. IMO this is not going to cause any discernable real world issues

Reply:
Great no probs 👍